components/Componenten/search-bar/search-bar.njk

```diff
@@ -1,4 +1,4 @@
-<div class="dso-search-bar">
+<div {{ className('dso-search-bar', [modifier, modifier]) }}>
```

Comment:
Volgens mij is de array notatie niet nodig.. Met `className('dso-search-bar', modifier)` komt er ook nette code uit als `modifier` falsy is. Maar dat weet ik niet zeker

Reply:
aangepast